keyboards/kbd4x/readme.md

```diff
@@ -0,0 +1,15 @@
+# KBD4x
+
+![KBD4x](https://cdn.shopify.com/s/files/1/1473/3902/products/1_ef8d3ad0-fc70-4d4f-9e36-8438c781a581_1800x1800.jpg)
+
+A compact 40% (12x4) ortholinear keyboard kit sold by KBDFans.
+
+Keyboard Maintainer: ![sevenseacat](https://github.com/sevenseacat)
```

Comment:
Lines 7 and 8 both need to end with two spaces so the file is rendered properly when viewed.